### PR TITLE
Add ReadMe

### PR DESCRIPTION
**Pull request by javiermtzo99**
Create ReadMe File

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+The data set that was used for the analysis of this project was created by Jack W Smith, JE Everhart, WC Dickson, WC Knowler, RS Johannes. The data set was sourced from the National Library of Medicine database from the National Institues of Health. Access to their respective analysis can be found [here](https://pmc.ncbi.nlm.nih.gov/articles/PMC2245318/) and access to the dataset can be found via [kaggle](https://www.kaggle.com/datasets/uciml/pima-indians-diabetes-database/data). Each row/obersvation from the dataset is an individual that identifies to be a part of the Pima (also known as The Akimel O'odham) Indeginous group, located mainly in the Central and Southern regions of the United States. Each observation recorded has summary statistics regarding features that include the Age, BMI, Blood Pressure, Number of Pregnancies, as well as The Diabetes Pedigree Function (which is a score that gives an idea about how much correlation is between person with diabetes and their family history).
+
+## Report
+
+The final report can be found [here](https://github.com/UBC-MDS/diabetes_predictor_py/tree/main/analysis).
```
Comment (kuo4230) [inline README.md:15]: I think we might want to update this to the html link once we have the whole report ready to be exported as html? we can update this tomorrow before deadline.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+- conda (version 23.9.0 or higher)
+- conda-lock (version 2.5.7 or higher)
+- mamba (version 1.5.8 or higher)
+- jupyterlab (version 4.0.0 or higher)
+- nb_conda_kernels (version 2.3.1 or higher)
+- Python and packages listed in [`environment.yml`](https://github.com/UBC-MDS/diabetes_predictor_py/blob/main/environment.yml)
+
+## License
+
+The Breast Cancer Predictor report contained herein are licensed under the [Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0) License](https://creativecommons.org/licenses/by-nc-nd/4.0/) See the [license file](https://github.com/UBC-MDS/diabetes_predictor_py/blob/main/LICENSE.md) for more information. If re-using/re-mixing please provide attribution and link to this webpage. The software code contained within this repository is licensed under the [MIT license](https://opensource.org/license/MIT). See the [license file](https://github.com/UBC-MDS/diabetes_predictor_py/blob/main/LICENSE.md) for more information.
```
Comment (kuo4230) [inline README.md:46]: hmmm i think it's the diabetes report? lol

**Review by kuo4230 — CHANGES_REQUESTED** [review]
hey Javier, all looks good to me, we can maybe do some touchup later if any of us have time. just a couple things to update otherwise looks good to go.

**Comment by JennyonOort** [comment]
The original write has been merged to main branch so I am closing down the duplicate.

**Pull request closed without merging**